machine_learning/benchmarks/circuit_qnn_classifier_fit_benchmark.py

```diff
@@ -80,6 +77,55 @@ def parity(x):
             neural_network=self.circuit_qnn, optimizer=self.optimizers[optimizer_name]
         )
 
+    def setup_dataset_iris(self, X, quantum_instance_name, optimizer_name):
+        """Training CircuitQNN function for iris dataset."""
+
+        self.output_shape = 3
+
+        # scaling data
+        scaler = MinMaxScaler((-1, 1))
+        self.X = scaler.fit_transform(X)
+
+        # creating feature map
+        feature_dim = self.X.shape[1]
+        feature_map = ZFeatureMap(feature_dim)
```

Comment:
Same question, why `ZFeatureMap` not `ZZ`?

Reply:
Updated so both synthetic and iris dataset models use ZZFeatureMap